HiP-FeatureToggle/Controllers/FeatureGroupsController.cs

```diff
@@ -0,0 +1,125 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using PaderbornUniversity.SILab.Hip.FeatureToggle.Managers;
+using PaderbornUniversity.SILab.Hip.FeatureToggle.Models.Entity;
+using PaderbornUniversity.SILab.Hip.FeatureToggle.Models.FeatureGroups;
+using PaderbornUniversity.SILab.Hip.FeatureToggle.Services;
+using PaderbornUniversity.SILab.Hip.Webservice;
+using System;
+using System.Linq;
+
+namespace PaderbornUniversity.SILab.Hip.FeatureToggle.Controllers
+{
+    /// <summary>
+    /// Provides methods to add/remove feature groups and to assign users to these groups.
+    /// </summary>
+    [Authorize]
+    [Route("api/[controller]")]
+    public class FeatureGroupsController : Controller
+    {
+        private readonly FeatureGroupsManager _manager;
+        private readonly CmsService _cmsService;
+
+        public FeatureGroupsController(FeatureGroupsManager manager, CmsService cmsService)
+        {
+            _manager = manager;
+            _cmsService = cmsService;
+        }
+
+        [HttpGet]
+        public IActionResult GetAll([FromQuery]string identity)
```

Comment:
I think in that case the `identity` referes to the user that you want the information for, not the identity of the caller. Or did you mean that here, too?